src/dlsproto/client/legacy/internal/connection/DlsRequestConnection.d

```diff
@@ -417,7 +418,7 @@ public class DlsRequestConnection :
 
     override protected void handleNone ( )
     {
-        assert(false, "Handling command with code None");
+        enforce(false, "Handling command with code None");
```

Comment:
Can this ever happen? I notice that in dhtproto I left this as an `assert(false)`.

Reply:
I need to check swarm for that I guess. I'll see.

Reply:
It's hard to track this down, but I would say this _might_ be possible - in case `this.reader` reads `0` into `this.cmd`: https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/node/connection/ConnectionHandler.d#L711-L715

Reply:
Right, that's what I wondered. It looks to me like there should be an `enforce` at that point in swarm (reading command code 0 is invalid), but we can do this in *proto right now. No need to mess with the core of legacy swarm.